DataFormats/CTPPSAlignment/src/RPAlignmentCorrectionsData.cc

```diff
@@ -58,15 +60,18 @@ RPAlignmentCorrectionData RPAlignmentCorrectionsData::GetFullSensorCorrection(un
   bool useRPErrors) const
 {
   RPAlignmentCorrectionData c;
```

Comment:
Using a single letter variable name is discouraged by the CMSSW code rules. I know that it was already as such also in the previous version of it, but I would profit of this opportunity to give it a more descriptive name.